packages/components/src/tooltip/style.scss

```diff
@@ -17,9 +17,6 @@
 	font-size: 12px;
 	box-shadow: none;
 
-	// This prevents quickly hovering the tooltip from blocking hovering something else.
-	pointer-events: none;
```

Comment:
@jasmussen do you remember why we add this? (It was added on the original G2 branch).

Reply:
From the comment, it looks like it was added to make the tooltip "invisible" to a cursor so a user wouldn't accidentally click it when trying to click something else in the UI or on the canvas.

Reply:
I think Michael is right, yes. If you hover a button and see a tooltip, that might cover a button immediately above it, and if you quickly click there to click that button, you should be able to.